README.md

```diff
@@ -29,6 +29,19 @@ Update `prometheus/prometheus.yml` with the targets (server you wish to monitor)
 ./start-all.sh
 ```
 
+### Load original data to prometheus server
```

Comment:
Thanks, please added this info to the README itself, so users will make the connection, and can go back to Scylla wiki. Its not obvious to everyone